python-ci/pipeline.yaml

```diff
@@ -0,0 +1,37 @@
+env:
+  PYTHON_VERSION: "3"
+
+steps:
+  - label: ":pip: Install dependencies"
+    key: "pip"
+    command: pip install -r requirements.txt
+    plugins:
+      - docker#v5.9.0:
+          image: "python:${PYTHON_VERSION}"
+
+  - label: Lint with Ruff
```

Comment:
Looks like we might need a new [emoji](https://github.com/buildkite/emojis) added for [Ruff](https://docs.astral.sh/ruff/).

Reply:
https://github.com/buildkite/emojis/pull/523